src/main/resources/application-development.properties

```diff
@@ -9,12 +9,14 @@ spring.datasource.username=sa
 spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
 spring.h2.console.enabled=true
-app.showSwaggerUILink=true
+#app.showSwaggerUILink=true
```

Comment:
Please remove this commented out line of code
see; https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

otherwise, LGTM